src/porepy/models/verification_setups/terzaghi.py

```diff
@@ -0,0 +1,819 @@
+        out.numerical_consolidation_degree = self.numerical_consolidation_degree(
+            out.numerical_displacement,  # displacement
+            out.numerical_pressure,  # pressure
+        )
+        out.exact_consolidation_degree = self.exact_sol.consolidation_degree(t)
+
+        # Store errors data
+        out.pressure_error = self.relative_l2_error(
+            sd,  # grid
+            out.exact_pressure,  # true_array
+            out.numerical_pressure,  # approximate_array,
+            True,  # is_scalar
+            True,  # is_cc
+        )
+        out.consolidation_degree_error = np.abs(
+            out.exact_consolidation_degree - out.numerical_consolidation_degree
+        )
+
+        return out
+
+
+class SetupUtilities:
+    """Mixin class containing useful utility methods for the setup."""
+
+    params: dict
+    """Setup parameters dictionary."""
+
+    mdg: pp.MixedDimensionalGrid
+    """Mixed-dimensional grid. Only one subdomain for this verification."""
+
+    time_manager: pp.TimeManager
+    """Time-stepping object."""
+
+    stress_keyword: str
+    """Key for accessing data parameters for the mechanics subproblem."""
+
+    bc_values_mechanics_key: str
+    """Key for accessing mechanical boundary values."""
+
+    solid: pp.SolidConstants
+    """Solid constant object."""
+
+    fluid: pp.FluidConstants
+    """Fluid constant object."""
+
+    exact_sol: ExactSolution
+    """Exact solution object."""
+
+    results: list[SaveData]
+    """List of :class:`SaveData` objects containing the results of the verification."""
+
+    # ----> Derived physical quantities
+    def confined_compressibility(self) -> number:
+        """Compute confined compressibility [Pa^-1].
+
+        Returns:
+            Confined compressibility.
+
+        """
+        mu_s = self.solid.shear_modulus()
+        lambda_s = self.solid.lame_lambda()
+        m_v = 1 / (2 * mu_s + lambda_s)
+
+        return m_v
+
+    def consolidation_coefficient(self) -> number:
+        """Compute consolidation coefficient [m^2 * s^-1].
+
+        Returns:
+            Coefficient of consolidation.
+
+        """
+        k = self.solid.permeability()  # [m^2]
+        mu_f = self.fluid.viscosity()  # [Pa * s]
+        rho = self.fluid.density()  # [kg * m^-3]
+        gamma_f = rho * pp.GRAVITY_ACCELERATION  # specific weight [Pa * m^-1]
+        hydraulic_conductivity = (k * gamma_f) / mu_f  # [m * s^-1]
+        storage = self.solid.specific_storage()  # [Pa^-1]
+        if not storage == 0.0:
+            raise ValueError("Terzaghi's solution requires zero specific storage.")
+        alpha_biot = self.solid.biot_coefficient()  # [-]
+        m_v = self.confined_compressibility()  # [Pa^-1]
+
+        c_v = hydraulic_conductivity / (gamma_f * (storage + alpha_biot**2 * m_v))
+
+        return c_v
+
+    # ----> Non-dimensionalization methods
+    def nondim_time(self, t: number) -> number:
+        """Non-dimensionalize time.
```

Comment:
Non-dimensional or non-dimensionalized. Applies throughout.

Reply:
This meant to refer to the verb in infinite form, e.g., like _Compute time._

Reply:
Fixed.

Reply:
Ah, sorry.